test/rust_src/src/windows-tests.el

```diff
@@ -152,3 +152,30 @@
     (should (set-window-fringes w1 1 1 nil))
     (should-not (set-window-fringes w1 1 1 nil))
     (should (set-window-fringes w1 1 2 nil))))
+
```

Comment:
Bravo!

Reply:
Thanks! Note that the tests don't cover the `set_prevent_redisplay_optimizations_p` and `set_suspend_auto_hscroll` side effects, because I don't know what those mean...